quinn-proto/src/connection/streams.rs

```diff
@@ -164,14 +179,16 @@ impl Streams {
         // Revert to initial state for outgoing streams
         for dir in Dir::iter() {
             for i in 0..self.next[dir as usize] {
-                self.send.remove(&StreamId::new(self.side, dir, i)).unwrap();
+                let id = StreamId::new(self.side, dir, i);
+                self.send.remove(&id).unwrap();
```

Comment:
Since this also seems to free up streams, does it also have to increment the limit via `stream_freed`?
Or am I understanding something wrong regarding zero-rtt here?

Reply:
0-RTT streams are always locally-initiated (necessarily, since we haven't received anything from the peer yet) so we never need to issue flow control updates to account for them being open or closed, even if they're bidirectional. Added a comment to make it a bit clearer that skipping the `stream_freed` call is intentional.